src/elements/content-sidebar/activity-feed/comment/Comment.scss

```diff
@@ -124,7 +124,6 @@
         }
 
         .bcs-comment-content {
-            color: $fours;
```

Comment:
Is changing the non-task comment styles intentional?

Reply:
Yes, the designs are being updated to keep them in sync